### ISSUE TITLE
update discovery to include "showAssigned"

### ISSUE THREAD

**Issue by int-maxd** [googleapis/google-api-python-client#2447]
The Google Tasks API v1 has been updated with the ability to view assigned tasks from Spaces or Google Docs/Sheets under **assignmentInfo** in the task resource when returned from the API.

To pull these tasks, the flag **_"showAssigned"_** query parameters needs to be set. Currently the **tasks.v1.json**file in the discovery cache can be updated to accomodate the query parameter by adding the **_"showAssigned"_** definition under **resources > tasks > methods > list**.

Example of definition required: 

```json
"resources": {
    "tasks": {
        "methods": {
            "list" : {
                "showAssigned": {
                  "description": "Flag indicating whether assigned tasks are returned in the result. Optional. The default is False.",
                  "location": "query",
                  "type": "boolean"
            },
         }
      }
    }
}
```

**Comment by ohmayr** [comment]
@int-maxd It seems the discovery artifact for `tasks` in the cached directory was updated a month ago already includes `showAssigned`.

See: [tasks.v1.json#L421](https://github.com/googleapis/google-api-python-client/blame/3427189dfd410694b8590c7ada89b2f9dc41bce5/googleapiclient/discovery_cache/documents/tasks.v1.json#L421). This was released in `google-api-python-client==2.136.0` .

Can you try the following and see if resolves the issue for you?

```
pip install google-api-python-client==2.136.0
```

**Comment by ohmayr** [comment]
Closing this issue as per previous comment.
